sympy/parsing/autolev/__init__.py

```diff
@@ -0,0 +1,94 @@
+from sympy.external import import_module
+
+
+def parse_autolev(inp, output=None):
+    """Parses Autolev code to SymPy code.
+
+    Parameters
+    ----------
+    inp: str
+         1. Can be the name of a file containing Autolev code.
+         2. Can be a string containing Autolev code.
+
+    output: str
+            1. Can be the name of an output file to which the SymPy code should be written to.
+            2. Can be the string "print". In this case the SymPy code is written to stdout.
+            3. Can be the string "list". In this it returns a list containing the SymPy code.
+                Each element in the list corresponds to one line of code.
+
+    Example (Double Pendulum)
+    -------
+    >>> from sympy.parsing.autolev import parse_autolev  # doctest: +SKIP
+    >>> l = []  # doctest: +SKIP
+    >>> l.append("MOTIONVARIABLES' Q{2}', U{2}')"  # doctest: +SKIP
+    >>> l.append("CONSTANTS L,M,G)"  # doctest: +SKIP
+    >>> l.append("NEWTONIAN N)"  # doctest: +SKIP
+    >>> l.append("FRAMES A,B)"  # doctest: +SKIP
+    >>> l.append("SIMPROT(N, A, 3, Q1))"  # doctest: +SKIP
+    >>> l.append("SIMPROT(N, B, 3, Q2))"  # doctest: +SKIP
+    >>> l.append("W_A_N>=U1*N3>)"  # doctest: +SKIP
+    >>> l.append("W_B_N>=U2*N3>)"  # doctest: +SKIP
+    >>> l.append("POINT O)"  # doctest: +SKIP
+    >>> l.append("PARTICLES P,R)"  # doctest: +SKIP
+    >>> l.append("P_O_P> = L*A1>)"  # doctest: +SKIP
+    >>> l.append("P_P_R> = L*B1>)"  # doctest: +SKIP
+    >>> l.append("V_O_N> = 0>)"  # doctest: +SKIP
+    >>> l.append("V2PTS(N, A, O, P))"  # doctest: +SKIP
+    >>> l.append("V2PTS(N, B, P, R))"  # doctest: +SKIP
+    >>> l.append("MASS P=M, R=M)"  # doctest: +SKIP
+    >>> l.append("Q1' = U1)"  # doctest: +SKIP
+    >>> l.append("Q2' = U2)"  # doctest: +SKIP
+    >>> l.append("GRAVITY(G*N1>))"  # doctest: +SKIP
+    >>> l.append("ZERO = FR() + FRSTAR())"  # doctest: +SKIP
+    >>> l.append("KANE())"  # doctest: +SKIP
+    >>> l.append("INPUT M=1,G=9.81,L=1)"  # doctest: +SKIP
+    >>> l.append("INPUT Q1=.1,Q2=.2,U1=0,U2=0)"  # doctest: +SKIP
+    >>> l.append("INPUT TFINAL=10, INTEGSTP=.01)"  # doctest: +SKIP
+    >>> l.append("CODE DYNAMICS() double_pendulum.c)"  # doctest: +SKIP
+    >>> parse_autolev("\\n".join(l), "print")  # doctest: +SKIP
```

Comment:
Any reason this can't be written like:

```pycon
>>> """\
... My
... Autolev
... code\
... """
...
```
```

?

Reply:
It can be written in a better way. I shall change it.